pypi_formula_mappings.json

```diff
@@ -486,6 +486,9 @@
   "virtualenv": {
     "exclude_packages": ["six"]
   },
+  "volatility": {
+    "extra_packages": ["capstone", "pycryptodome", "yara-python"]
```

Comment:
it needs to exclude jsonschema too

Reply:
i don't think so, since in this case `jsonschema` is one of the [extra packages](https://github.com/volatilityfoundation/volatility3/blob/52f325ed54abfdd5d4a35067b7e281fec7881fa4/requirements.txt#L17-L18) we are adding

Reply:
But if we use the formula, don't we need it in `exclude_packages`?

Reply:
No, before I added the formula I manually added it in `extra_packages`. So depending on the formula instead just means removing it from `extra_packages`. Note, it is a bit confusing since upstream's [`requirements.txt`](https://github.com/volatilityfoundation/volatility3/blob/52f325ed54abfdd5d4a35067b7e281fec7881fa4/requirements.txt) are not the published dependencies, but rather [`requirements-minimal.txt`](https://github.com/volatilityfoundation/volatility3/blob/52f325ed54abfdd5d4a35067b7e281fec7881fa4/requirements-minimal.txt):

```
$ curl -fsSL https://pypi.org/pypi/volatility3/2.0.0/json | jq -r .info.requires_dist[]
pefile (>=2017.8.1)
```

In other words, running `brew update-python-resources volatility` on this pr results in no changes as expected